_data/cryptocurrencies.yml

```
@@ -552,6 +552,14 @@ websites:
       - sms
     doc: https://support.viabtc.com/hc/en-us/articles/900001693343
 
+  - name: Voyager
+    url: https://investvoyager.com
+    img: investvoyager.svg
+    tfa:
+      - totp
+      - sms
+    doc: https://www.investvoyager.com/blog/introducing-2fa/
```

```suggestion
    doc: https://support.investvoyager.com/support/solutions/43000367274
```

I reckon that this page is a more useful doc